### DISCUSSION TITLE
`helmfile.yaml.gotmpl` isn't detected by helmfile manager

### DISCUSSION THREAD

**Discussion by z0rc** [renovatebot/renovate#23164] [Suggest an Idea]
### How are you running Renovate?

Mend Renovate hosted app on github.com

### If you're self-hosting Renovate, select which platform you are using.

None

### Was this something which used to work for you, and then stopped?

I am trying to get this working for the first time

### Describe the problem

Helmfile is moving from `helmfile.yaml` to `helmfile.yaml.gotmpl`. Reasoning why described at https://github.com/helmfile/helmfile/blob/main/docs/proposals/towards-1.0.md#force-gotmpl-file-extension-for-helmfileyaml-in-case-you-want-helmfile-to-render-it-as-a-go-template-before-yaml-parsing. Right now helmfile v0 accepts both .yaml and .yaml.gotmpl files (https://github.com/helmfile/helmfile/blob/main/pkg/app/app.go#L1213), this will change to just yaml.gotmpl with helmfile v1.

Minimal reproduction: https://github.com/z0rc/renovate-helmfile-yaml-gotmpl

### Relevant debug logs

<details><summary>Logs</summary>

```
Copy/paste the relevant log(s) here, between the starting and ending backticks
```

</details>

### Have you created a minimal reproduction repository?

I have linked to a minimal reproduction in the description above

**Comment by rarkins** [comment]
This is a feature request, not a problem

**Reply by z0rc** [reply]
Up to you to reclassify it.

**Comment by pathob** [comment]
Implemented with #26063. Can be closed.
